.travis.yml

```diff
@@ -1,15 +0,0 @@
-language: R
```

Comment:
Let's keep this file in for now, it's ok if it keeps failing on travis we will eventually get it to work